package.json

```diff
@@ -19,7 +19,7 @@
   ],
   "scripts": {
     "clean": "mkdir -p lib/ && rm -rf lib/",
-    "build": "gulp build",
+    "build": "gulp build --silent",
```

Comment:
it copies the static test fixtures over, as well as generating the json schema from typescript types.